### PR TITLE
:arrow_up: Adding scaling section

### PR DESCRIPTION
**Pull request by GianfrancoBazzani**
Hi guys, this PR adds a new section named Scalability. Added some notes on that and left few subsections to be completed. Let me know if there is any issue ^^.

### CODE REVIEW

**Comment by taxmeifyoucan** [comment]
Thanks a lot for the contribution! I am thrilled to see the first page coming from community. I left a review, sorry if I was nitpicking too much but we can move it forward :)

**Comment by GianfrancoBazzani** [comment]
Sorry @taxmeifyoucan i can't se your comments.

**Review by taxmeifyoucan — CHANGES_REQUESTED** [review]

**Comment by taxmeifyoucan** [comment]
@GianfrancoBazzani  Sorry I didn't finish the review. Also I'd like to see other people interested in the same topic to chime in, feel free to share it in discord :)

File: docs/wiki/protocol/scalability.md [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+- Data availability: Data availability mechanics that ensures that the data is available to be queried when it is required.
+- Settlement: Mechanics that guarantee the finality and immutability of transactions.
+
+Modular blockchain approach suggests the separation of a blockchain system into different logical layers which are designed to perform well as a few specific functional components. The result is a scalable blockchain architecture with simpler and more efficient modular components, rather than a large and complex monolithic blockchain that cannot scale effectively.
+
+## Scaling Ethereum
+The core Ethereum protocol (Layer 1) is continuously developed by the community through EIPs, which open up the possibility of introducing changes to the base protocol to enhance network scalability.  Furthermore, Ethereum's high level of programmability makes it possible to create scalable solutions on top of the platform (Layers 2).
+
+### Ethereum Layer 1 scaling
+#### Split consensus from execution (The Merge).
```
[the remark this answers is not in the document]
Comment (GianfrancoBazzani) [reply docs/wiki/protocol/scalability.md]: Which section do you think that will be the correct place be for the merge event? maybe The Protocol - History?

File: docs/wiki/protocol/scalability.md [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+> **we can define the scalability of a specific blockchain as its capacity to perform well under an increase in user transaction demands or/and in the volume of users**.
+
+## Blockchain Scalability Limits
+Consensus mechanics require most of validator nodes in the network to publicly agree on the next valid block. This directly affects the block latency, which is the time it takes for a new block to be created and validated. The block latency and block size directly determine the transactional throughput of the blockchain, which is measured by the **TPS** (transactions per second) metric.  Block latency can be influenced by a variety of factors, such as the computational power of the network nodes, the complexity of the consensus mechanism, the network traffic and the block size. Hence, a network made up of a few high-performance nodes tightly connected to each other and using large block sizes can result in an exceptional blockchain with high TPS and remarkably low block latency. However, such a network would likely be highly centralized and require users to make significant trust assumptions.
```
[the remark this answers is not in the document]
Comment (GianfrancoBazzani) [reply docs/wiki/protocol/scalability.md]: I re-writed a bit the section to be more Ethereum specific. LMK if there's any issue.

File: docs/wiki/protocol/scalability.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (GianfrancoBazzani) [reply docs/wiki/protocol/scalability.md]: I moved it to The Protocol - History

**Pull request merged**